packages/react-components/react-charts-preview/src/components/HorizontalBarChart/HorizontalBarChart.tsx

```diff
@@ -28,14 +28,13 @@ export const HorizontalBarChart: React.FunctionComponent<IHorizontalBarChartProp
   let _barHeight: number;
   //let _classNames: IProcessedStyleSet<IHorizontalBarChartStyles>;
   let _uniqLineText: string = '_HorizontalLine_' + Math.random().toString(36).substring(7);
-  let _calloutId: string;
+  let _calloutId: string = useId('_callout');
```

Comment:
move this id setting within popover. I see same parameter being used everywhere

Reply:
moved